cMake/SetupSLEPC.cmake

```diff
@@ -0,0 +1,42 @@
+# Locate SLEPC include paths and libraries
+# SLEPC can be found at https://gitlab.com/slepc/slepc
+
+# This module defines
+# SLEPC_INCLUDE_DIRS, where to find SLEPC.h, etc.
+# SLEPC_LIBRARIES, the libraries to link against to use SLEPC.
+# SLEPC_FOUND, If false, don't try to use SLEPC.
+
+function(ConfigureSLEPC TARGET)
+  message(STATUS "Looking for SLEPC")
+
+  # FIXME: windows paths were not tested
+  FIND_PATH(SLEPC_INCLUDE_PATH
+    NAMES slepc.h
+    PATHS
+    ${CMAKE_INCLUDE_PATH}
+    "[HKEY_CURRENT_USER\\software\\slepc\\include]"
+    "[HKEY_CURRENT_USER\\slepc\\include]"
+    $ENV{SLEPCROOT}/include/
```

Comment:
Arch linux adds a env-variable which can be parsed:

```
$ cat /etc/profile.d/slepc.sh
export SLEPC_DIR=/opt/slepc/linux-c-opt
```

Reply:
Added `$ENV{SLEPC_DIR}/include/`. Please, check that `$ENV{SLEPC_DIR}/lib/` points to the .so file.